fundamentals/examples.tex

```diff
@@ -151,79 +151,111 @@ \subsection{Simulations}
+        reactor was also configured to take in either enriched UOX fuel or
+        recycled MOX fuel - preferring the latter if available.
+
+    \item \textbf{Recycled Fuel Fabrication} (\Class{cycamore::Fab}): This
+        facility requests depleted uranium and separated fissile material and
+        mixes them together into recycled MOX fuel that it offers to
+        requesters.  The streams are mixed using a D-factors approach
+        \TODO{add D-factors citation} in an attempt to match simple neutronics
+        properties of an ideal target fuel composition.
+
+    \item \textbf{Separations} (\Class{cycamore::Separations}): This facility
+        takes in all kinds of spent fuel and separates it into plutonium and
+        waste streams with some efficiency (0.99 was used for these
+        simulations).  Up to 60,000 kg of Pu can be separated per month.
+
+    \item \textbf{Repository} (\Class{cycamore::Sink}): This is an
+        infinite-capacity facility that can take in all types of material
+        including separations waste streams and spent reactor fuel.
+
+    \item \textbf{UOX Fabrication} (\Class{cycamore::Source}): This facility
+        provides enriched UOX fuel as requested with infinite capacity.
+
+    \item \textbf{DU Source} (\Class{cycamore::Source}): This facility provides
+        depleted uranium as requested with infinite capacity.
+
+\end{itemize}
 
-The \Class{Source} and \Class{Sink} archetypes from \Cycamore have been used as a
-source of fresh, enriched uranium and a repository, respectively. A simple
-custom recycled fuel fabrication facility was also used, which mixes fissile 
-and fertile streams to match target fuel compositions as closely
-as possible.  The method used for this matching is approximately the same as
-that used by the \gls{COSI} fuel cycle simulator \TODO{cite COSI equivalence
-method}. The material flows for this simulation are shown below in Figure
-\ref{fig:flowmodopen}.
+For the once through case, the \Class{Reactor} was set to offer all spent fuel
+as waste.  For the modified open case, the \Class{Reactor} offerred spent UOX
+into a spent fuel market where it is preferrentially sent to separations, but
+spent MOX is still offerred as waste.  In the closed case, the \Class{Reactor}
+offers all spent fuel into a spent fuel market. Because the separations
+facility requests spent fuel with a higher preference than the repository, it
+is sent there if possible.  All these preferences are easy to adjust and
+\Cyclus will dynamically handle supply constraints and non-uniform preference
+resolution.  It is notable that separations and recycle fuel fabrication
+capacity is deployed identically in all simulations.  In the once-through
+case, the recycling loop never acquires material, and so reactors always
+receive fresh UOX fuel.  In the modified open case, the recycle loop only
+acquires spent UOX.  The \gls{DRE} ensures everything operates smoothly in all
+cases.
```

Comment:
This paragraph is all over the place. Each sentence seems address a completely different topic. Please provide a topic sentence and support it with more detail. Then conclude.